Snek.cs

```diff
@@ -1,131 +1,132 @@
+using System.ComponentModel.DataAnnotations;
 using System.Numerics;
 using System.Reflection;
 using System.Runtime.CompilerServices;
 using Raylib_cs;
 
 namespace Snek;
 
+public struct Cord
+{
+    public int X;
+    public int Y;
+
+    public readonly bool Equals(Cord other)
```

Comment:
Talk to me on this thread how you arrived at this method and what it problem it solves? 

Do you have any insight about any general patterns that can emerge in other situations that are similar?